snippets/perl/get_nb_jobs.pl

```diff
@@ -0,0 +1,46 @@
+#Load module netbackup.pm from current directory
+use lib '.';
+
+use netbackup;
+use Getopt::Std;
+
+my %args;
+getopt('supdt', \%args);
+my $fqdn_hostname = $args{s};
+my $username = $args{u};
+my $password = $args{p};
+my $domainname = $args{d};
+my $domaintype = $args{t};
+
+if (!$fqdn_hostname || !$username || !$password) {
+  print "\nUsage : perl get_nb_jobs.pl -s <master_server> -u <username> -p <password> [-d <domain_name>] [-t <domain_type>]\n\n";
+  die;
+}
+
+print "\nRecieved the following parameters : \n";
+print " FQDN Hostname : $fqdn_hostname\n";
+print " Username : $username\n";
+print " Password : $password\n";
+if ($domainname) {
+  print " Domain Name : $domainname\n";
+}
+if ($domaintype) {
+  print " Domain Type : $domaintype\n";
+}
+
+if ($domainname && $domaintype) {
+  $myToken = netbackup::login($fqdn_hostname, $username, $password, $domainname, $domaintype);
+}
+else{
+  $myToken = netbackup::login($fqdn_hostname, $username, $password);
+}
+
+#print "Login token: $myToken\n";
```

Comment:
Remove it and other commented code lines.

Reply:
done